jwst/tso_photometry/tso_photometry.py

```diff
@@ -146,23 +146,23 @@ def tso_aperture_photometry(datamodel, xcenter, ycenter, radius, radius_inner,
             del int_num, mid_utc
             nrows = 0                   # flag as bad
         else:
-            log.debug("Times are from the INT_TIMES table.")
+            log.debug("Times are from the INT_TIMES table")
             time_arr = mid_utc[offset: offset + num_integ]
             int_times = Time(time_arr, format='mjd', scale='utc')
 
+    # compute integration time stamps on the fly
     if nrows == 0:
-        log.debug("Times were computed from EXPSTART and TGROUP.")
-
-        dt = (datamodel.meta.exposure.group_time *
-              (datamodel.meta.exposure.ngroups + 1))
-        dt_arr = (np.arange(1, 1 + datamodel.meta.exposure.nints) *
-                  dt - (dt / 2.))
+        log.debug("Times computed from EXPSTART and EFFINTTM")
+        dt = datamodel.meta.exposure.integration_time
+        n_dt = (datamodel.meta.exposure.integration_end -
+                datamodel.meta.exposure.integration_start + 1)
```

Comment:
Tests are failing here because one of these keywords is None.

Reply:
https://travis-ci.org/github/spacetelescope/jwst/jobs/725671432#L982

Reply:
Yep. Fixed in next commit. The test was missing integration_end.